docs/data.rst

```diff
@@ -275,6 +275,35 @@ are ``[{"x":100, "y":0}]``, but the contents of ``my_point_array`` are:
 ``[{"x":100, "y":1}, {"x":2, "y":200}, {"x":3, "y":3}]``.
 
 
+Accessing complex members
```

Comment:
I think we should split this info in the three existing sections (structs, sequences and arrays, unions) instead of creating a new one.